app/unsubscribe_requests/rest.py

```diff
@@ -0,0 +1,15 @@
+from flask import Blueprint, current_app, jsonify
+from datetime import datetime
+from app.dao.unsubscribe_request_dao import get_count_of_unprocessed_requests
+
+
+
+unsubscrube_requests_blueprint = Blueprint("unsubscribe_requests", __name__)
+
+@unsubscrube_requests_blueprint.route("/unsubscribe/<uuid:notification_id>/summary", methods=["GET"])
+def unsubscribe_requests_summary(notification_id):
+    pending_unsubscribe_requests = 250
+    pending_unsubscribe_requests = get_count_of_unprocessed_requests(notification_id)
+    datetime_of_latest_unsubscribe_request = datetime.now()
+    return jsonify(pending_unsubscribe_requests=pending_unsubscribe_requests, datetime_of_latest_unsubscribe_request=datetime_of_latest_unsubscribe_request), 200
```

Comment:
`count_of_pending_unsubscribe_requests` might be better (consistent with `datetime_of_latest_unsubscribe_request`):
```suggestion
    return jsonify(count_of_pending_unsubscribe_requests=pending_unsubscribe_requests, datetime_of_latest_unsubscribe_request=datetime_of_latest_unsubscribe_request), 200
```